api/src/main/java/org/apache/gravitino/job/SparkJobTemplate.java

```diff
@@ -30,6 +30,21 @@
  * Represents a job template for executing Spark applications. This class extends the JobTemplate
  * class and provides functionality specific to Spark job templates, including the class name, jars,
  * files, archives, and configurations required for the Spark job.
+ *
+ * <p>Take Spark word count job as an example:
+ *
+ * <p>className: "org.apache.spark.examples.JavaWordCount" executable
+ * "https://example.com/spark-examples.jar" arguments: ["{{input_path}}", "{{output_path}}"]
```

Comment:
so the `executable` is the actual job resource, not the engine (such as spark, bash, etc.) to execute the job?

If the answer is yes, how does the user specify engine information, such as the Spark version or Spark home?

Reply:
That needs to be configured in the server side, like specifying the Spark home. I will do this when I implement the Spark executor.